h/static/scripts/vendor/diff_match_patch_uncompressed.js

```diff
@@ -1,2200 +0,0 @@
-/**
```

Comment:
Yay! No more patched vendor libs. We'll have to see about restoring show differences without this.